tools/intersmash-tools-core/src/main/java/org/jboss/intersmash/tools/junit5/IntersmashExtension.java

```diff
@@ -106,11 +98,6 @@ public void beforeAll(ExtensionContext extensionContext) throws Exception {
 			}
 		} catch (Throwable t) {
 			tt = Optional.of(t);
-			//			try {
```

Comment:
same here ...

Reply:
See above, isn't it removed already? Or am I missing what you mean here?